ports/aom/vcpkg.json

```
@@ -1,10 +1,10 @@
 {
   "name": "aom",
   "version-semver": "3.2.0",
-  "port-version": 1,
+  "port-version": 2,
   "description": "AV1 codec library",
   "homepage": "https://aomedia.googlesource.com/aom",
-  "supports": "!uwp & !arm",
+  "supports": "!(windows & arm & !uwp)",
```

```suggestion
  "supports": "!(windows & arm64)",
```
So this should be `!(windows & arm64)`?

---

this would prevent arm64-uwp to build, but actually arm64-uwp builds just fine

---

I think the only configuration that would fail are arm-windows and arm64-windows. arm-uwp/arm64-uwp are ok. so `"!(windows & arm & !uwp)"` should be correct

---

`!(windows & arm & !uwp)` is `!windows | !arm | uwp` -- is it really intended to disable `arm-windows` but enable `arm-uwp`?